diesel/src/lib.rs

```diff
@@ -93,7 +93,7 @@
 //! You can come ask for help at
 //! [gitter.im/diesel-rs/diesel](https://gitter.im/diesel-rs/diesel)
 
-#![cfg_attr(feature = "unstable", feature(specialization, try_from))]
+#![cfg_attr(feature = "unstable", feature(specialization))]
```

Comment:
We should remove the gating on the uses of this feature as well (grep `try_from`)

Reply:
try_from has been stabilized (ungated) at nightly but it is still nightly only API.

Reply:
Ah I see

Reply:
Can you update the FIXME comment to specify which Rust version we can use it?